### ISSUE TITLE
[Question] How to interpret Prophet regression coefficients?

### ISSUE THREAD

**Issue by rohan-gt** [facebook/prophet#1972]
Assume I'm building 2 regression models on the same data (Sales ~ Price):
1. Default Linear Regression model (without standardization) with Price beta B1
2. Default Prophet model with additional regressor Price with beta B2

I want to make the regression coefficients from the two models similar in interpretation. I'm aware that the Prophet model standardizes the data before building the model. I want to know what transformation I need to perform to B2 to make it similar to the interpretation of B1. [This](https://github.com/facebook/prophet/issues/598) issue mentions setting **standardize=False** but I wanted a more generic interpretation and comparison of the coefficients from the two models.

**Comment by tcuongd** [comment]
Hey there, we recently added utility functions to return the coefficients of the extra regressors on the original scale of the data. See https://facebook.github.io/prophet/docs/seasonality,_holiday_effects,_and_regressors.html#coefficients-of-additional-regressors

Let me know if you have any questions :)

**Comment by rohan-gt** [comment]
@tcuongd I'm aware of the utility functions. I was asking about the derivation of the coefficients from the utility functions and how they compare to the linear regression coefficients. Do they have the same interpretation since some standardization of the target variable happens internally in the case of Prophet

**Comment by tcuongd** [comment]
Yes the output of the utility functions is on the scale of the original data. Here is the derivation from the source code:

https://github.com/facebook/prophet/blob/b75844e07c3b09bf3bc383c9d43241c554dd251b/python/prophet/utilities.py#L28-L77

**Comment by tcuongd** [comment]
Hey @rohan-gt did you have any further questions about this?

**Comment by rohan-gt** [comment]
Hi @tcuongd, what is the difference in to interpretation of the coefficients when you turn the standardization on vs off and which one of these aligns more with the simple linear regression coefficients? Also you can't turn off the standardization for the y variable in prophet right? How does that affect the interpretation of it's coefficients?

**Comment by CJRoelofs** [comment]
Hi, is it possible to restrict the coefficients of the additional regressors to be negative?
